lib/nexmo/oas/renderer/presenters/content_switcher.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Nexmo
+  module OAS
+    module Renderer
+      module Presenters
+        class ContentSwitcher
+          def initialize(format:, label:, force_type: nil, theme_light:)
```

Comment:
can you place `force_type` at the end of the list? It's the only one that has a default value